pkg/controller/template/render_test.go

```diff
@@ -429,6 +439,11 @@ func TestGenerateMachineConfigs(t *testing.T) {
 		if !foundKubeletUnitWorker {
 			t.Errorf("Failed to find kubelet unit for worker")
 		}
+		if foundIPForwarding != shouldFindIPForwarding {
+			t.Errorf("Expected IP forwarding file mismatch. Was rendered: %t, Was expected: %t",
```

Comment:
it covers two cases, one where we expect the file to be there because its an OVN config being rendered and the other case with SDN where we expect the file not to be there. I could just drop the word "Expected" if that makes more sense?

Reply:
yes let's drop "expected" makes it weird in the test when we are looking for the failure..